server/services/homekit/lib/sendState.js

```diff
@@ -26,6 +26,15 @@ function sendState(hkAccessory, feature, event) {
         );
       break;
     }
+    case `${DEVICE_FEATURE_CATEGORIES.MOTION_SENSOR}:${DEVICE_FEATURE_TYPES.SENSOR.BINARY}`: {
```

Comment:
Same as buildService file, you can use LEAK_SENSOR or OPENING_SENSOR case

Reply:
Fixed